### PR TITLE
Fixes magfest/ubersystem/issues/2766: Custom room assignment email for MAGLabs

### PR DESCRIPTION
**Pull request by RobRuana**

### CODE REVIEW

**Comment by RobRuana** [comment]
Okay, I'm gonna commit this one right now, because Josiah needs it before this evening. But please feel free to review and suggest changes.

File: magclassic/templates/emails/room_assignment.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+This year in particular has been difficult for {{ c.EVENT_NAME }} Staff Rooms. If your room assignment isn't like what you've requested, please understand that your patience with the rough rooming situation helps out {{ c.EVENT_NAME }} immensely. We've had to cut room nights for many staff on Thursday and Sunday nights. These cuts were discussed with department heads, who felt that it was better to cut room nights than run the risk of going over budget. That being said, if there is an irreconcilable issue with your room assignment, please let us know so we can adjust your room assignment.
+
+{% if room.message %}{{ room.message }}
+
+{% endif %}Again, PLEASE - Make sure that checking into your room is the first thing you do when you get onsite. If you have problems in that process, come to {{ c.EVENT_NAME }} Staff Operations Department to get it resolved. If there are any roommates related issues with your room at event, please come to Staff Operations and let one of the Operations Staff know. We will work with the hotel liaison team in an attempt to resolved your issue as soon as possible. We appreciate your hard work and we hope that you enjoy {{ c.EVENT_NAME }}.
```
Comment (kitsuta) [inline magclassic/templates/emails/room_assignment.txt]: Typo here -- "resolved" when it should be "resolve."

Comment (RobRuana) [inline magclassic/templates/emails/room_assignment.txt]: I'm just gonna change that on master, cool?

Comment (RobRuana) [inline magclassic/templates/emails/room_assignment.txt]: Also needs to be fixed in the uber repo...

Comment (kitsuta) [inline magclassic/templates/emails/room_assignment.txt]: Yup, that's fine. :)

**Pull request merged**